### DISCUSSION TITLE
Placement of copyright statements TMW vs mh_style/mh_copyright

### DISCUSSION THREAD

**Discussion by drittelhacker** [florianschanda/miss_hit#257] [Q&A]
Is there any information about the placement of the copyright notice that I have overlooked? I mean; where in the file it has to be such that `mh_style` and `mh_copyright`actually find it? 

Since `help myfun` prints the first comment block in `myfun.m`, we decided to add a separate comment right after that for the copyright statement. We are aware that this is against standard recommendations, but it is still pretty much at the top of the file, and we get the correct help text printed. 

However, both mh-tools don't like this style, and I am looking for a way to get this resolved that is better than ignoring `mh_copyright` and banning the copyright rule for `mh_style`. Any suggestions or recommendations?

Also, when running `mh_copyright --add-notice` on a file, the only thing `help` gives you afterwards is the copyright statement. Are we being too old-school, documenting our functions this way? Or is this actually worth an issue?

**Comment by florianschanda** [comment]
So there are two places MH looks:
* the absolute top of the file
* the top-level comment in a function

So top-level being: there are no other MATLAB statements at all before that comment.

As to recommendations, things are flexible, it is only SW after all :) Can you open a ticket with an example of how you'd like it and I will see what I can do...

**Reply by drittelhacker** [reply]
I have just done so. See, #259
I hope the description makes clear enough what we want.
